client/src/dataset/DatasetFunctions.js

```diff
@@ -45,4 +45,19 @@ function getDatasetAuthors(dataset) {
     : null;
 }
 
-export { mapDataset, getDatasetAuthors };
+/**
+ * Display a file with some metadata. Has the following parameters:
+ *
+ * @param {Object[]} images - Images link arrays
+ */
+function getDatasetImageUrl(images) {
+  try {
+    return images?.length > 0 ?
+      images[0]["_links"][0].href : undefined;
+  }
+  catch {
+    return undefined;
+  }
```

Comment:
```suggestion
  try {
    return images[0]["_links"][0].href;
  }
  catch {
    return undefined;
  }
```
Testing for the length of the first array seems unnecessary since we rely on `try/catch` for all the other unexpected cases (and the `getDatasetImageUrl` function is always invoked after a `images?.length > 0 ?` in the code).